pkg/workflows/workflows.go

```diff
@@ -0,0 +1,49 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+// Package workflows gives utility functions related to REANA's workflows.
+package workflows
```

Comment:
The package name `workflows` seems "important", but its contents seems "underused". Might be better called `workflowutils`, but that's weird.  We can keep it as it, and think whether it wouldn't grow in importance, say `workflows/list.go`, `workflows/status.go`, etc to hold more of the "core" business logic?

Reply:
If we're planning to extend the package that way, should we at least rename the file to `utils.go`, without changing the package name?